src/views/pages/address/enter-address-form.njk

```diff
@@ -3,167 +3,84 @@
-            },
-            {
-                value: "GB-ENG",
-                text: i18n.address.enterAddress.countryEngland
-            },
-            {
-                value: "GB-SCT",
-                text: i18n.address.enterAddress.countryScotland
-            },
-            {
-                value: "GB-WLS",
-                text: i18n.address.enterAddress.countryWales
-            },
-            {
-                value: "GB-NIR",
-                text: i18n.address.enterAddress.countryNorthernIreland
-            }]
-        }) }}
-        </fieldset>
-
-        {{ govukInsetText({
-        html: "<h2 class='govuk-heading-m'>" + publicInformationTitle | escape + "</h2>
-            <p>" + publicInformationLine1 | escape + "</p>"
-        }) }}
-
-        {% include "includes/save-and-continue-button.njk" %}
+{% include "includes/validators.njk" %}
+
+<div>
+    {{ govukInput({
+        label: {
+            text: i18n.address.enterAddress.propertyNameOrNumber,
+            classes: "govuk-!-margin-top-5"
+        },
+        hint: {
+            text: i18n.address.enterAddress.propertyNameOrNumberHint
+        },
+        id: "premises",
+        name: "premises",
+        value: address.premises,
+        attributes : {  
+            required : true,
+            oninvalid: validateNameOrNumber,
+            oninput: validateNameOrNumber
+        }
+    }) }}
+
+    {{ govukInput({
+        label: {
+            text: i18n.address.enterAddress.addressLine1
+        },
+        id: "address_line_1",
+        name: "address_line_1",
+        value: address.address_line_1,
+        attributes : {
+            required : true,
+            oninvalid: validateAddressLine1,
+            oninput: validateAddressLine1
+        }
+    }) }}
+
+    {{ govukInput({
+        label: {
+            text: i18n.address.enterAddress.addressLine2
+        },
+        id: "address_line_2",
+        name: "address_line_2",
+        value: address.address_line_2,
+        attributes : {  
+            oninput: "validateLength(this, 50, '" + i18n.address.enterAddress.errorMessages.addressLine2Length | escape + "');"            }
+    }) }}
+
+    {{ govukInput({
+        label: {
+            text: i18n.address.enterAddress.townOrCity
+        },
+        classes: "govuk-!-width-two-thirds",
+        id: "locality",
+        name: "locality",
+        value: address.locality,
+        attributes : {
+            required : true,
+            oninvalid: validateTownOrCity,
+            oninput: validateTownOrCity
+        }
+    }) }}
+
+    {{ govukInput({
+        label: {
+            text: i18n.address.enterAddress.county
+        },
+        classes: "govuk-!-width-two-thirds",
+        id: "region",
+        name: "region",
+        value: address.region,
+        attributes : {
+            oninput: "validateLength(this, 50, '" + i18n.address.enterAddress.errorMessages.countyLength | escape + "');"
+        }
+    }) }}
+</div>
```

Comment:
this file should be renamed as it no longer includes the <form , perhaps just name it enter-address-fields.njk